indexer/services/vulcan/src/lib/on-message.ts

```diff
@@ -86,7 +98,7 @@ export async function onMessage(message: KafkaMessage): Promise<void> {
     const handler: Handler = new (getHandler(update))!(
       getTransactionHashFromHeaders(message.headers),
     );
-    await handler.handleUpdate(update);
+    await handler.handleUpdate(update, message.headers ?? {});
 
```

Comment:
can we also emit a stat here comparing to message_received_timestamp ?

Reply:
we already do that [here](https://github.com/dydxprotocol/v4-chain/pull/1237/files#diff-fbcd2ef7f5fc44a4c9b34e080e07494f74a92010e4ef52fa449a1c6c98fe3cdcR65-R76) in the same code block right?

Reply:
that's before handling the message though